src/admin/components/forms/Form/generateArray.tsx

```diff
@@ -0,0 +1,32 @@
+import { useWatchForm } from "./context";
```

Comment:
I forgot to delete this import, I was trying to generate the arrays in this file and then export them, but I got an invalid hook call.